go-update/src/main/java/org/reactome/release/goupdate/DuplicateReporter.java

```diff
@@ -64,10 +66,11 @@ public DuplicateReporter(MySQLAdaptor adaptor)
 	/**
 	 * Gets the number of referrers for each instance of a duplicated accession.
 	 * @param accession - The accession to look up.
+	 * @param classesToIgnore - A list of class names to ignore, when looking for referrers. Any referrer whose Schema Class is in classes to ignore will not be considered as a referrer, and will not be added to the count.
 	 * @return The DB_IDs of the duplicated accession mapping to the number of referrers of each one.
 	 * @throws Exception 
 	 */
-	public Map<Long, Integer> getReferrerCountForAccession(String accession) throws Exception
+	public Map<Long, Integer> getReferrerCountForAccession(String accession, String ... classesToIgnore) throws Exception
```

Comment:
If you decide to use the `getGOInstancesByAccession` method in DuplicateCleaner, you could re-use it here and do something like this (with the `getGOInstancesByAccession` living whereever is appropriate):
```
public Map<Long, Integer> getReferrerCountForAccession(String accession, String ... classesToIgnore) throws Exception
{
	List<GKInstance> instances = DuplicateCleaner.getGOInstancesByAccession(this.adaptor, accession);

	return instances
		.stream()
		.collect(
			toMap(
				GKInstance::getDBID,
				i -> getReferrerCountforInstance(i, classesToIgnore)
			)
		);
}
```